### PR TITLE
First commit

### PR DESCRIPTION
**Pull request by MAIK-01**

### CODE REVIEW

File: src/makeRobotAccountant.js [hunk 1]
```diff
@@ -6,7 +6,21 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  return (x) => {
+    return (y) => {
+      if (count++ > 2) {
```
Comment (Bohdan-Maksymiuk) [inline src/makeRobotAccountant.js]:
```suggestion
      if (counter > 3) {
```

Comment (Bohdan-Maksymiuk) [reply src/makeRobotAccountant.js]: don't use increment in such way

File: src/makeRobotAccountant.js [hunk 2]
```diff
@@ -6,7 +6,21 @@
...
+        if (count % 2 === 0) {
```
Comment (Bohdan-Maksymiuk) [inline src/makeRobotAccountant.js]: no need for 2 if's
just join them with logic `AND` - &&

**Review by Bohdan-Maksymiuk — CHANGES_REQUESTED** [review]

File: src/makeRobotAccountant.js [hunk 3]
```diff
@@ -6,7 +6,18 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  return (x) => {
+    return (y) => {
+      count++;
+      if (count > 2 && count % 2 === 0) {
```
Comment (ArtemTopolskyi) [inline src/makeRobotAccountant.js:14]:
```suggestion
      count++;
    
      if (count > 2 && count % 2 === 0) {
```

**Review by ArtemTopolskyi — APPROVED** [review]

**Pull request closed without merging**